### ISSUE TITLE
reads counting

### ISSUE THREAD

**Issue by capricy** [Xinglab/rmats-turbo#189]
Hi rMATS support team,

I noticed that MATS paper described to only use the uniquely mapped reads for the splicing event modeling. I wonder if the rMATS follows the same idea?

Thanks.

C.

**Comment by EricKutschera** [comment]
Yes rMATS requires reads to be uniquely mapped (NH = 1): https://github.com/Xinglab/rmats-turbo/blob/v4.1.2/rMATS_pipeline/rmatspipeline/rmatspipeline.pyx#L279

This post has some other details about rMATS read filtering: https://github.com/Xinglab/rmats2sashimiplot/issues/33#issuecomment-656720560
